build.gradle

```diff
@@ -52,6 +52,8 @@ def junitJupiterVersion = dependencyManagement.importedProperties['junit-jupiter
 dependencies {
 
   compile(
+      "uk.q3c.rest:hal-kotlin:0.5.3.2.a87244c",
+      'com.squareup.okhttp3:okhttp:3.13.1',
```

Comment:
Hvorfor er denne her?

Reply:
Mange av testene feiler for både meg og Espen hvis denne ikke er med.

Reply:
Kan den være en testavhengighet da?

Reply:
Testet nå, hvis jeg tar den i testCompile så vil ikke testene gå gjennom.

Reply:
Dette er fikset nå